src/activities/menu/ConfigurationItem.qml

```diff
@@ -404,6 +413,58 @@ Item {
                 }
             }
         }
+        
+        Flow {
```

Comment:
You're right, maybe we should change the behavior for wordset the same way:

If activated in config
    if rcc already downloaded (DownloadManager.haveLocalResource())
        update and/or register (DownloadManager.updateResource())
    else
        Popup dialog and ask for immediate download; if confirmed
            start download unconditionally (DownloadManager.downloadResource())
   
For backgroundMusic this would be triggered on enabledBackgroundMusic. For wordset (which is not your job for your pull request) on useExernalLargeWordset.

Alternatively we could skip the request dialog and simply download based on the automaticDownload setting.